### ISSUE TITLE
Stack' (a stack with bells and whistles galore)

### ISSUE THREAD

**Issue by danielmkarlsson** [tidalcycles/Tidal#274]
Wouldn't be nice if there was a thing that was similar to `stack`but played one stem at a time.

featureCreaping from that core idea: Wouldn't it be nice if you could express the amount of repeats of one stem stochastically (min 4 repeats, max 16 repeats).

If there were no end to featureCreap then I'm pretty sure we'd all benefit from the stack depleting all of it's stems once before starting over and no repeats at last stem, first stem of new stack start either.

**Comment by bgold-cosmos** [comment]
That sounds a lot like `seqP`, although it doesn't have the stochastic stuff.

**Comment by danielmkarlsson** [comment]
Ok sure thing. I am fine w/ calling this idea `seqP` with stochastic stuff.
Do you feel that would be a nice addition tho?

**Comment by vism2889** [comment]
Fond of this idea, would definitely make use of it.  It would make sense to me to apply `irand` or something to the second argument of one of the stems of `seqP`, or `seqPLoop`, except the next stem in the `seqP` would have a set start cycle still.  Would be cool to see a  `seqP'` or something that took two or three params/arguments, a priority param giving one stem favor over others in the `seqP'` stems, meaning this stem will be played more or less frequently (similar to the existing "degrading" pattern events functions like `sometimesBy`), the other two parameters would be ints to designate min and max repeats per stem.  

Don't know how to implement this, tried messing around with `irand`, `choose` and `sometimesBy` with the `seqP` stem values, i get the error: `couldn't match 'Pattern Integer' with 'Ratio Integer'`.

**Comment by yaxu** [comment]
Closing for now, I think `wrandcat` does the first one. Feel free to open a new issue with seqP enhancement ideas.
